### PR TITLE
Update pictogram guidance 

### PR DESCRIPTION
**Pull request by peter-garvin**
Updating pictogram guidance to be consistent with new guidance on IDL. I think I got all the images over but let's double check to make sure.

**This is a dependency of https://github.com/carbon-design-system/carbon/pull/8213**

Related PRs:
https://github.com/carbon-design-system/design-language-website/pull/874

https://github.com/carbon-design-system/carbon/pull/8213

Related issues:
https://github.com/carbon-design-system/design-language-website/issues/903

### CODE REVIEW

**Comment by peter-garvin** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by peter-garvin** [comment]
@vpicone @mattrosno Can you help me out here? Not sure why the deployment failed

File: src/pages/guidelines/pictograms/usage.mdx [hunk 1]
```diff
@@ -7,104 +7,172 @@ tabs: ['Library', 'Usage', 'Code', 'Contribute']
 ---
 
 import { Bee32 } from '@carbon/icons-react';
+import { noGutterSm } from '../../../styles/Grid.module.scss';
```
Comment (mattrosno) [inline src/pages/guidelines/pictograms/usage.mdx:10]: @peter-garvin this relative path might be wrong?

Comment (mattrosno) [reply src/pages/guidelines/pictograms/usage.mdx:10]: The Vercel errors are calling out incorrect paths to `Grid.module.scss`.

image

Comment (mattrosno) [reply src/pages/guidelines/pictograms/usage.mdx]: I don't know that IDL website code base, but that line was attempting to use a generated class name from a file that only lives in the IDL website. The build should pass by removing that line, unless there were other issues as well.

**Comment by peter-garvin** [comment]
OK I think everything looks good on my end. Thanks for the help @mattrosno

**Comment by peter-garvin** [comment]
I cant add a ready for review tag, but this is ready for review

**Review by jillianhowarth — APPROVED** [review]
Looks good to me! 🎉

**Review by emyarod — APPROVED** [review]
looks good to me

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Comment by peter-garvin** [comment]
@tw15egan when can this be updated on the site?

**Comment by tw15egan** [comment]
@peter-garvin let's get it live now 👍🏻

**Pull request merged**